vsix/Standalone/source.extension.vsixmanifest

```diff
@@ -14,10 +14,24 @@
     <Tags>WinRT, C++, cppwinrt, native</Tags>
   </Metadata>
   <Installation AllUsers="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,17.0)" />
-  </Installation>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.0, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>  </Installation>
```

Comment:
Was `</Installation>` meant to go on a new line here?

Reply:
In the VSIX manifest for VS 2017+, there's a block like this (in this example it's supposed to install on VS 2017 or VS 2019):

```
    <Installation>
        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,17.0)" />
        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Community" />
    </Installation>
```

To make it support VS 2022 as well, my example needs to be (as I understand it)

```
    <Installation>
        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,17.0)" />
        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Enterprise" />
        <InstallationTarget Version="[15.0,17.0)" Id="Microsoft.VisualStudio.Community" />
        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0, 18.0)">
            <ProductArchitecture>amd64</ProductArchitecture>
        </InstallationTarget>
        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0, 18.0)">
            <ProductArchitecture>amd64</ProductArchitecture>
        </InstallationTarget>
        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0, 18.0)">
            <ProductArchitecture>amd64</ProductArchitecture>
        </InstallationTarget>
    </Installation>
```

Reply:
I meant stylistically, right now both `</InstallationTarget>` and `</Installation>` are on the same line